grails-app/services/org/pih/warehouse/data/ProductAssociationDataService.groovy

```diff
@@ -24,16 +25,30 @@ class ProductAssociationDataService {
         log.info "Validate data " + command.filename
 
         command.data.eachWithIndex { params, index ->
+            DateFormat dateFormat = new SimpleDateFormat("yyyy-mm-dd hh:mm:ss")
+            if (params['dateCreated']) params['dateCreated'] = dateFormat.parse(params['dateCreated'])
+            if (params['lastUpdated']) params['lastUpdated'] = dateFormat.parse(params['lastUpdated'])
+
+            params['code'] = params['code']?.toUpperCase()
```

Comment:
Why there is `toUpperCase`?

Reply:
as mentioned by Katarzyna after QA
_"when try to import association type "Substitute" or "substitute", works only for SUBSTITUTE - it shouldn't matter and can be confusing for users. "_

Reply:
ok